gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -128,15 +115,30 @@ fluid.defaults("gpii.flowManager.local", {
             }
         }
         */
-        // TODO: After FLUID-6148 is implemented, directly saving "defaultLifecycleInstructions" into options should
-        // be replaced by expressing {that}.defaultLifecycleInstructions as an expander in the component's options
-        "userLogonHandling.defaultLifecycleInstructions": {
-            record: "{that}.options.defaultLifecycleInstructions",
-            target: "{that gpii.lifecycleManager.userLogonHandling.stateChangeHandler}.options.defaultLifecycleInstructions"
+        defaultSettingsDataPromise: "@expand:fluid.promise()"
+    },
+    contextAwareness: {
+        suppressHttpEndpoints: {
+            checks: {
+                addHttpEndpointsOption: {
+                    contextValue: "{gpii.flowManager.local}.options.suppressHttpEndpoints",
```

Comment:
I wonder why this ended up as an addition to this diff, which doesn't seem relevant

Reply:
This part is irrelevant and stays unchanged. I guess moving the big comment block that explains "defaultSnapshot" and "defaultLifecycleInstructions" up above confuses the github diff.